### ISSUE TITLE
Ability to upload designs, compressed with zip

### ISSUE THREAD

**Issue by esl51** [avocode/avocode#1251]
It would be cool if it was the ability to upload zip files with designs. For example, PSD files very well compressed.

**Comment by vojtatranta** [comment]
Hello @esl51,
size of PSD is not the bottleneck of the upload process. Yes, zip could save around 20% but zip still must be unpacked on the server and psd must be parsed in its full size. So overall saving would not be big, maybe few seconds.

**Comment by jankuca** [comment]
I think they are referring to the upload speed (bad connection?), not the speed of processing.

**Comment by esl51** [comment]
Yes, I mean upload speed. PSD sometimes compressed to 50 and more percents, and the Internet connection is poor. As for the processing of designs, they can be unpacked after uploading automatically and then have to work as usual.
